website/src/docs/react-initializing.md

```diff
@@ -10,54 +10,28 @@ category: "React"
 
 When using Uppy's React components, an Uppy instance must be passed in to the `uppy={}` prop from the outside. This Uppy instance must be initialized before passing it to the desired component, and be cleaned up using `uppy.close()` when you are done with it.
 
-## Functional Components
+## Functionl Components
```

Comment:
Shouldn't this be "Functional Components"?

Reply:
oops yes, I was going back and forth between "Function Components" and "Functional Components" and I guess i ended up in the middle 🤣 
```suggestion
## Functional Components
```